Engineering-Reference/M365DestinationAO.md

```diff
@@ -284,95 +280,87 @@ Default Value: 1800000
 
 The timeout for operations with the server.
 
-Change Conditions: ??
+Change Conditions: Timeout value can be changed. Can cause performance issues, not recommended.
 
 ### Maximum Results Per Request <a name="maxresreq"></a>
 {: .no_toc }
 Default Value: 999
 
 The maximum number of results to return for individual queries.
 
-Change Conditions: ??
+Change Conditions: Value can be changed. Can cause performance issues, not recommended.
 
 ### Provision Office 365 Group Timeout Check <a name="prov365time"></a>
 {: .no_toc }
 Default Value: 600000
 
 The maximum period of time used to check if an Office 365 Group has been created.
 
-Change Conditions: ??
+Change Conditions: Timeout value can be changed. Can cause performance issues, not recommended.
 
 ### Default Document Library Name <a name="defdoclib"></a>
 {: .no_toc }
 Default Value: Documents
 
-The default timezone to use with appointments when migrating to Exchange 2007 when the timezone is not identified by other means.
-
-Change Conditions: ??
+When migrating Microsoft Teams and Microsoft Groups and the source platform is not Office 365 or SharePoint On Premise, documents will be migrated to this library
 
 ### Teams Direct Migration <a name="teamdirmig"></a>
 {: .no_toc }
 Default Value: Off
 
 If the Teams site does not exist in the destination, keep this off so the migration uses a Prestage Teams location. This is a faster migration and more user friendly. If the destination Team site already exists, turn this on to prevent an error from occuring.
 
-Change Conditions: ??
-
 ### Migrate Team Channel Tabs <a name="migteamtab"></a>
 {: .no_toc }
 Default Value: Off
 
 Migrate Team channel tabs such as: Web, Document (Word, Excel, PDF, Powerpoint). For more information, see article 'Team Channel Tabs'
 
-Change Conditions: ??
+Change Conditions: Disable to prevent migration of tabs.
 
 ### Retry Count <a name="tretryc"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: 10
 
 The number of times an operation will be attempted before failing.
 
-Change Conditions: ??
+Change Conditions: Value can be changed. Can cause performance issues, not recommended.
 
 ### Create Office 365 Group <a name="crea365grou"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: On
 
 Create Office 365 Group if it does not exist.
 
-Change Conditions: ??
+Change Conditions: Turn off to not create groups.
```

Comment:
Turning this off will have the tool not create any Microsoft Groups that do not exist. <Warning> If a group is on the source but not on the destination and this is turned off the migration will fail.

Reply:
done